### ISSUE TITLE
question about .train()

### ISSUE THREAD

**Issue by ForawardStar** [zhuoinoulu/pidinet#43]
I notice that in the function train() of main.py , the model are converted to training mode through model.train(). But, batch normalization or DropOut is not adopted in the network. I wonder is it necessary to use model.train(). Can I abandon it?
